src/main/java/com/softserve/actent/verification/service/SendEmail.java

```diff
@@ -0,0 +1,61 @@
+package com.softserve.actent.verification.service;
+
+import com.softserve.actent.model.entity.User;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Properties;
+
+@Slf4j
+@Service
+public class SendEmail {
+
+    public boolean sendSimpleEmail(String email, User user){
+
+        String confirmUrl = "http://localhost/confirm?login=" + user.getLogin() + "&uuid=" + user.getUuid();
+
+        final Properties properties = new Properties();
+        try {
+            properties.load(SendEmail.class.getClassLoader().getResourceAsStream("mail.properties"));
+        } catch (IOException e) {
+            log.error("Cant download main.properties file");
+        }
+
+        Session session = Session.getInstance(properties, new javax.mail.Authenticator(){
+            @Override
+            protected PasswordAuthentication getPasswordAuthentication() {
+                return new PasswordAuthentication("onlyTwo2111@gmail.com", "qwert1234asd");
```

Comment:
Pure strings, especially passwords, should be declared as constants anywhere else.

Reply:
.